feeds/ucentral/uspot/files/usr/bin/captive

```diff
@@ -17,6 +17,68 @@ function interfaces() {
 	return uniq(interfaces);
 }
 
+function generate_spotfilter(name) {
+	function parse_bool(val) {
+		if (val == "1" || val == "on" || val == "true" || val == "yes")
+			return true;
+		else if (val == "0" || val == "off" || val == "false" || val == "no")
+			return false;
+		else
+			return null;
+	};
+	function fail(msg) {
+		warn(msg + '\n');
+		exit(1);
+	}
+
+	if (!parse_bool(uci.get('uspot', name, 'configure_spotfilter')))
+		exit(0);
+
+	let uspot = uci.get_all('uspot', name);
+	if (!uspot)
+		fail('Cannot load uspot config for "' + name);
+
+	let device_macaddr = uci.get('network', uspot.interface, 'device');
```

Comment:
Technically, this makes it possible to setup spotfilter redirect to an interface that isn't part of the same network as the wlan devices. Is this ok?
Shouldn't spotfilter be hooked to a network interface (as in, config/network interface) instead of specific devices?
It would then make it trivial to have uspot running on a router and have downstream dumb APs bridged to that interface.